indispensable-cosef/src/commonMain/kotlin/at/asitplus/signum/indispensable/cosef/CoseSignedBytes.kt

```diff
@@ -32,11 +32,12 @@ data class CoseSignedBytes(
 ) {
     fun toCoseSignatureInput(
         externalAad: ByteArray = byteArrayOf(),
```

Comment:
i know it is not new, but still: why is this not null?

Reply:
Quoting https://www.rfc-editor.org/rfc/rfc9052.html#section-4.4

> The externally supplied data from the application, encoded in a bstr type. If this field is not supplied, it defaults to a zero-length byte string.